tests/operator/operator.go

```diff
@@ -764,12 +764,15 @@ var _ = Describe("[Serial][sig-operator]Operator", Serial, decorators.SigOperato
 			return vmis
 		}
 
-		startAllVMIs = func(vmis []*v1.VirtualMachineInstance) {
-			for _, vmi := range vmis {
-				vmi, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi, metav1.CreateOptions{})
+		startAllVMIs = func(vmis []*v1.VirtualMachineInstance) []*v1.VirtualMachineInstance {
```

Comment:
Since you basically re-write this function, would you consider making it a static `func` with zero indentation and no duplicate signature? Redundant lamdas are harmful.

Reply:
Sure.

Reply:
Done